As of mongodb-8.0.32, the public-release config no longer disables the remote cache.
Switch to local config but add the release tag/stamp and remove the jobs=auto override.

diff '--color=auto' -ur a/.bazelrc b/.bazelrc
--- a/.bazelrc	2026-09-11 15:16:39.000000000 -0400
+++ b/.bazelrc	2026-09-17 15:36:41.697664965 -0400
@@ -449,8 +449,9 @@
 common:local --remote_cache_compression=false
 common:local --grpc_keepalive_time=0s
 common:local --legacy_important_outputs
-common:local --jobs=auto
+common:local --//bazel/config:release=True
 common:local --//bazel/config:local_build=True
+common:local --stamp
 
 
 # Disable remote execution but keep remote cache enabled
